packages/share/lib/share.dart

```diff
@@ -43,4 +44,17 @@ class Share {
 
     return channel.invokeMethod('share', params);
   }
+
+  static Future<void> shareFile(Uri uri, String mimeType) {
```

Comment:
`Uri`s may refer to many things that are not files. Could we either rename to `shareUri` or change the type of the first argument to `File` and then use the `File.uri` getter when constructing the `params` map below?